docs: update commands with base-uri additional information (#2221)
treeverse/lakeFS: docs/reference/commands.md

```

### lakectl base-uri

> Base URI value is a lakefs reference or part of a reference that starts lakectl uses for each argument that accept reference and doesn't start with `lakefs://` prefix.

Reviewer: ```suggestion
Base URI allows you to use relative paths. When setting the Base URI, lakectl will add the Base URI as a prefix to all relative paths (paths not starting with lakefs://).
```

> The value for base URI is set by a flag `--base-uri` and the default value when not set is the set by an environment variable LAKECTL_BASE_URI.

Reviewer: ```suggestion
The value for base URI is set by the flag `--base-uri`. the default value could be set by the environment variable LAKECTL_BASE_URI.
```

> Having the environemnt varilabe sets to a repository, for example, can shorten your command by passing relative path:

Reviewer: ```suggestion
Setting the environment variable to a repository will allow you to use only the relative path, enabling you to shorten your commands, for example:
```
